src/com/palmergames/bukkit/towny/war/flagwar/listeners/TownyCommonListener.java

```diff
@@ -0,0 +1,29 @@
+package com.palmergames.bukkit.towny.war.flagwar.listeners;
+
+import com.palmergames.bukkit.towny.TownySettings;
+import com.palmergames.bukkit.towny.object.Translation;
+import com.palmergames.bukkit.towny.war.common.events.WarPreUnclaimEvent;
+import com.palmergames.bukkit.towny.war.flagwar.FlagWar;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+
+/**
+ * Listener for processing events from Towny's Common War Events.
+ */
+public class TownyCommonListener implements Listener {
+
+  @EventHandler (priority= EventPriority.HIGH)
+  private void onWarPreUnclaim(WarPreUnclaimEvent event) {
+    if (FlagWar.isUnderAttack(event.getTown()) && TownySettings.isFlaggedInteractionTown()) {
+      event.setCancelMessage(Translation.of("msg_war_flag_deny_town_under_attack"));
+      event.setCancelled(true);

```

Comment:
On a previous change (maybe neutrality toggling) I put in returns after the event.setCancelled(true), just in case this first statement and the following statement were both true, the 2nd cancellation message would override the first.

Reply:
I don't think it was the Neutrality toggles, but I see what you're saying there.